.circleci/config.yml

```diff
@@ -9,24 +9,41 @@ jobs:
     steps:
       - checkout
       - run: cargo fetch
+      - save_cache:
+          key: ds-v1-docker--{{ arch }}-{{ .Branch }}-{{ epoch }}
```

Comment:
If you use something like `{{ checksum "Cargo.lock" }}`, which is the only reason that the cached dependencies will change, you'll reduce cache thrashing even more 😸

Reply:
Yep, done.